src/controller/content/dto/content.dto.ts

```diff
@@ -166,14 +166,74 @@ export class chapterDto {
 
 }
 
-export class quizDto {
+export class quizVariantDto {
   @ApiProperty({
-    type: String,
-    example: 'What is the national animal of India',
+    type : [Object],
+    example: [
+      {
+          "blocks": [
+              {
+                  "key": "fbh77",
+                  "text": "What is Python",
+                  "type": "unstyled",
```

Comment:
Why are we storing the values for type, depth, inline style ranges, entity ranges, and data?
@sukanyamohanty143 @Arunesh700